vcsh

```diff
@@ -65,10 +66,12 @@ source_all() {
 
 # Read configuration and set defaults if anything's not set
 [ -n "$VCSH_DEBUG" ]                  && set -vx
-: ${XDG_CONFIG_HOME:="$HOME/.config"}
+: "${XDG_CONFIG_HOME:="$HOME/.config"}"
```

Comment:
This seems to leave `$HOME/.config` unquoted

Reply:
Ah, wait, maybe it does not.

Reply:
Same as [here](https://github.com/RichiH/vcsh/pull/293#discussion_r607069450).